src/baxter_action_node.cpp

```diff
@@ -0,0 +1,32 @@
+// Node that runs Baxter action servers using the control_msgs API.
+
+#include "controller_manager_msgs/ListControllers.h"
+#include "controller_manager_msgs/SwitchController.h"
+#include "rapid_pbd/action_names.h"
+#include "rapid_pbd/baxter_actions.h"
+#include "rapid_pbd_msgs/ArmControllerState.h"
```

Comment:
Remove includes for ListControllers, SwitchController, ArmControllerState, FreezeArm, and RelaxArm, which are unused.